<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="pokemonWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>521</height>
    </rect>
   </property>
   <widget class="QWidget" name="aiPage">
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>210</y>
       <width>82</width>
       <height>77</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="noAiButton">
        <property name="text">
         <string>1 v 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="oneAiButton">
        <property name="text">
         <string>1 v AI</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="twoAiButton">
        <property name="text">
         <string>AI v AI</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="playerChoicePage">
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>200</y>
       <width>160</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="newPlayerButton">
        <property name="text">
         <string>New</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadPlayerButton">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="battlePage"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
